cuda/apps/src/kinect_ransac.cpp

```diff
@@ -158,7 +158,7 @@ class SimpleKinectTool
     run (bool use_device)
     {
 #if 1
-      pcl::Grabber* filegrabber = 0;
+      pcl::Grabber* filegrabber;
```

Comment:
Any reason why you didn't remove this kind of variables? (unused, dead-stores)

Reply:
`      pcl::PCDGrabber<pcl::PointXYZRGB > filegrabber (path, frames_per_second, repeat);
`

PCDGrabber doesn't depend on Grabber?
Confused on this one.

Reply:
No, this variable is not needed, the one you pointed out is the actual variable shadowing this one.

Reply:
Okay :+1: 
I'll update.